internal/tiltfile/tiltfile_state.go

```diff
@@ -1480,6 +1485,14 @@ func (s *tiltfileState) imgTargetsForDepsHelper(mn model.ManifestName, imageMapD
 			WithTiltFilename(image.tiltfilePath).
 			WithImageMapDeps(image.imageMapDeps)
 
+		// once loaded to apiserver, the reconciler will take care of keeping
+		// this up-to-date, but we ensure the properties are initialized based
+		// on the values we've got so far
+		iTarget, err = iTarget.InferImagePropertiesFromCluster(s.defaultReg)
```

Comment:
this won't work correctly. 

`defaultReg` is the result of the default_registry function

in a real `docker-prune`, you need to read this off the cluster.

Reply:
(e.g., you want to prune `localhost:5000/my-image` on a kind cluster)